build.gradle

```diff
@@ -347,14 +354,14 @@ pmd {
     ruleSets = []
 }
 
-findbugs {
-    toolVersion = findbugsVersion
+spotbugs {
+    toolVersion = spotbugsVersion
     visitors = [
         "FindDeadLocalStores"
```

Comment:
Will the build fail if there are violations?

Reply:
Yes. I've overriden `equals()` without `hashCode()` ,then run `gradle lint`. Build failed as on image,
https://i.imgur.com/bFBlKix.png

Reply:
Maybe we should run all visitors? This param run only `FindDeadLocalStores` check. I havn't changed it from findBugs.

Reply:
When run lint it fail. https://imgur.com/a/stXsg

Reply:
> Maybe we should run all visitors? This param run only FindDeadLocalStores check. I havn't changed it from findBugs.

If you are interested, we can work together to see what rules we can enforce. The reason for only including one rule for findbugs is findbugs lacks a critical feature of printing violations in build log (the only way to see the violation is by examining xml or html). According to your screenshots, it seems like that spotbugs still cannot print violations to the build log :(. We need tackle this problem before we can enforce any rules. Nevertheless, let finish this PR first before we can discuss more.

Reply:
@xpdavid Maybe we can add more visitors by checking the current warnings given by findbugs.
Although since the intellij plugin isn't available,  we may have to force the developer to see the generated html report after running `gradlew lint` incasethe build fails

Reply:
I've found interesting plugin for gradle https://github.com/tomasbjerre/violations-gradle-plugin
It works with findBugs so with spotBugs problably too. Should I try to add it?